releasedir/fs_blobs_dir.go

```diff
@@ -300,6 +304,23 @@ func (d FSBlobsDir) UploadBlobs() error {
 	return nil
 }
 
+func (d FSBlobsDir) containsSymlinks() bool {
+	files, err := d.fs.RecursiveGlob(filepath.Join(d.dirPath, "**/*"))
+	if err != nil {
+		return false
+	}
+
+	for _, file := range files {
+		fileInfo, _ := d.fs.Lstat(file)
```

Comment:
@dpb587-pivotal originally I was thinking that unless I can verify the FileInfo of the symlink I cannot really react on the issue but you are right I modified the function to also return an error now and pass along the PathError if it exists.